stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -634,6 +634,8 @@ function(_compile_swift_files
   list(APPEND swift_flags "-enable-experimental-feature" "SE427NoInferenceOnExtension")
   list(APPEND swift_flags "-enable-experimental-feature" "AllowUnsafeAttribute")
   list(APPEND swift_flags "-enable-experimental-feature" "NonescapableTypes")
```

Comment:
Late to the party, but is there a reason why we kept this line?

Reply:
See my resolved comment: https://github.com/swiftlang/swift/pull/77637#discussion_r1844274996

Reply:
Ah, I see, thanks!